src/main/alphabetical.ts

```diff
@@ -0,0 +1,67 @@
+
```

Comment:
can we add this to a deeper directory? into assets/js or something

Reply:
good spot. I've moved both this and the other webpack JS file (index.js) that creates the bundles into a seperate folder